tests/cephfs/cephfs_utilsV1.py

```diff
@@ -2981,3 +2981,26 @@ def wait_for_service_to_be_in_running(self, client1, node):
         for service in services:
             if service["status"] != 1:
                 raise CommandFailed(f"service : {service['service_name']} is not UP")
+
+    def get_active_nfs(self, nfs_servers, virtual_ip):
```

Comment:
can the functions "get_active_nfs" and "get_active_nfs_server" be moved to utils? I see these 2 functions definition being added in all NFS tests which is not needed. Common modules to be reused and not include in every test.